### PR TITLE
Add Delete functionality.

### PR DESCRIPTION
**Pull request by difanyang**
The parameter of the JavaScript function may change depending on how the links are displayed.

### CODE REVIEW

File: easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting links. */
+@WebServlet("/delete-link")
+public class DeleteLinkServlet extends HttpServlet {
```
Comment (zixiyan) [inline easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java:15]: please do consider to put the deletion functionality into manageServlet.

File: easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
+    Key linkEntityKey = KeyFactory.createKey("Link", id);
+    DEFAULT_DATASTORE_SERVICE.delete(linkEntityKey);
```
Comment (xiaoluLucy814) [inline easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java:21]: I guess here you need to do something like `ServletHelper.DEFAULT_DATASTORE_SERVICE`

Comment (xiaoluLucy814) [reply easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java:21]: And also import the helper class

Comment (difanyang) [reply easylinks/src/main/java/com/google/step/servlets/DeleteLinkServlet.java]: Done.

**Pull request merged**